### PR TITLE
Create BPF_Door_port_redirect.yml

### PR DESCRIPTION
**Pull request by Zandmann**
BPFDoor ports redirect for evasion

### CODE REVIEW

**Review by Zandmann — COMMENTED** [review]
file renamed
MITRE edited

**Comment by frack113** [comment]
Hi, rule yml filename is lowercase

**Review by Zandmann — COMMENTED** [review]
Hi, changed to lowercase

**Review by Zandmann — COMMENTED** [review]
changed FP to capital letter

File: rules/linux/auditd/lnx_auditd_bpfdoor_port_redirect.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+title: BPFDoor_TCP_ports_redirect
```
Comment (nasbench) [inline rules/linux/auditd/lnx_auditd_bpfdoor_port_redirect.yml:1]: Can you please remove the `_` from the title and make every first letter a capital letter?

**Review by Zandmann — COMMENTED** [review]
changed

**Pull request merged**